src/System.AppContext/src/System.AppContext.csproj

```diff
@@ -10,6 +10,7 @@
     <!-- The following line needs to be removed once we have a targeting pack for 4.6.2 -->
     <TargetingPackNugetPackageId Condition="'$(TargetGroup)'=='net462'">Microsoft.TargetingPack.NETFramework.v4.6</TargetingPackNugetPackageId>
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">netstandard1.5</PackageTargetFramework>
+    <ExcludeResourcesImport Condition="'$(IsPartialFacadeAssembly)'=='true'">true</ExcludeResourcesImport>
```

Comment:
Default value for IsPartialFacadeAssembly is empty. I think the condition is correct assuming we only want to exclude the resources for the facade :)